### ISSUE TITLE
Golurk and Galvantula location.

### ISSUE THREAD

**Issue by sphynxkitten** [kwsch/PKHeX#2606]
I caught these two Pokemon in game and they're failing legality check due to "unable to match an encounter from original game" and thought I'd report it here so you could add the locations to the check.

Golurk:
Original game: Shield
Met location: at West Lake Axewell (in a Wild Area)
Met level: 40

Galvantula:
Original game: Shield
Met location: at South Lake Miloch (in a Wild Area)
Met level: 60

**Comment by sphynxkitten** [comment]
Hey kwsch, I noticed the locations were added, but not sure if Galvantula's met level was also added.

I noticed on Serebii [here](https://www.serebii.net/pokedex-swsh/galvantula/locations.shtml#shield), that it doesn't list level 60, but it does [here](https://www.serebii.net/pokearth/galar/southlakemiloch.shtml). 

In fact, it seems once you reach a certain point in the game, they're all available at a met level of 60.

Sorry and ignore this if it was already changed along with location, I just seen the location added aha.

![2019122916533000-B8FAEF4816CAC2B76D11869B05CA7601](https://user-images.githubusercontent.com/32999729/71553084-13c19400-2a5d-11ea-8f31-c008325d5e4a.jpg)
![2019122916534800-B8FAEF4816CAC2B76D11869B05CA7601](https://user-images.githubusercontent.com/32999729/71553085-191ede80-2a5d-11ea-90bb-22279f345a0c.jpg)
